tools/docker/besu-all-in-one/Dockerfile

```diff
@@ -1,29 +1,21 @@
-ARG BESU_VERSION=22.7.4
-ARG TESSERA_VERSION=22.1.7
+FROM hyperledger/besu:25.4.0
 
-FROM hyperledger/besu:$BESU_VERSION AS besu
-FROM quorumengineering/tessera:$TESSERA_VERSION AS tessera
+LABEL org.opencontainers.image.source="https://github.com/hyperledger-cacti/cacti"
 
-COPY --from=besu /opt/besu/ /opt/besu/
+RUN apt update -y && apt install -y curl
```

Comment:
Please add  `&& rm -rf /var/lib/apt/lists/*` and optionally `--no-install-recommends` to minimize image size

Reply:
@outSH Oops, thank you for catching that! I'll update it in a sec!

Reply:
@outSH Please re-review when you get the chance and let me know! Heads up: I've also fixed some of the failing tests that were dependent on the key files we were adding to the image earlier (now they just pull the key data directly from the genesis.json instead)